controllers/handler.go

```diff
@@ -133,7 +133,7 @@ func (ah *App) loginPOST(
 	validationErr := AccountDataToErrorMessage(accData)
 	if validationErr != nil {
 		w.WriteHeader(http.StatusUnauthorized)
-		sr.Message = validationErr.Error()
+		sr.Message = "Invalid username or password!"
```

Comment:
I'd rather we changed AccountDataToErrorMessage is a way that it does not check username and password separately. Like instead of two ifs we could do 1 if with an && (AND) condition and return this messaged when it's not met.

Reply:
can u do this? coz when i'm trying to change some lines on golang i feel like a potato :0